Geometry/DTGeometryBuilder/plugins/DTGeometryValidate.cc

```diff
@@ -82,11 +82,12 @@ class DTGeometryValidate : public one::EDAnalyzer<> {
 
 DTGeometryValidate::DTGeometryValidate(const edm::ParameterSet& iConfig)
     : dtGeometryToken_{esConsumes<DTGeometry, MuonGeometryRecord>(edm::ESInputTag{})},
-      infileName_(
-          iConfig.getUntrackedParameter<string>("infileName", "Geometry/DTGeometryBuilder/data/cmsRecoGeom-2021.root")),
+      //infileName_(iConfig.getUntrackedParameter<string>("infileName", "Geometry/DTGeometryBuilder/data/cmsRecoGeom-2021.root")),
       outfileName_(iConfig.getUntrackedParameter<string>("outfileName", "validateDTGeometry.root")),
       tolerance_(iConfig.getUntrackedParameter<int>("tolerance", 6)) {
-  fwGeometry_.loadMap(infileName_.c_str());
+  edm::FileInPath fp("Geometry/DTGeometryBuilder/data/cmsRecoGeom-2021.root");
```

Comment:
@slomeo , may be you can keep the `infileName` as a configurable parameter in validateDTGeometry_cfg.py and use
```
edm::FileInPath fp(infileName_.c_str());
```